src/clr-angular/data/datagrid/datagrid-column.ts

```diff
@@ -341,25 +378,38 @@ export class ClrDatagridColumn<T = any> extends DatagridFilterRegistrar<T, Datag
   }
 
   public get filterValue() {
-    return this.filter.value;
+    if (this.filter instanceof DatagridStringFilterImpl || this.filter instanceof DatagridNumericFilterImpl) {
+      return this.filter.value;
+    }
   }
 
   @Input('clrFilterValue')
-  public set updateFilterValue(newValue: string) {
+  public set updateFilterValue(newValue: string | [number, number]) {
```

Comment:
Note to self for future updates: we'll need to find a way to type the column so that we can infer the type of this setter based on the `colType` of the column.

Don't worry about it for this PR.